.github/workflows/healthchecks_azurekeyvault_secrets_ci.yml

```diff
@@ -0,0 +1,65 @@
+name: HealthChecks AzureKeyVault Secrets CI
+
+on:
+  workflow_dispatch:
+  push:
+   branches: [ master ]
+   paths:
+     - src/HealthChecks.AzureKeyVault.Secrets/**
+     - test/HealthChecks.AzureKeyVault.Secrets.Tests/**
+     - test/_SHARED/**
+     - .github/workflows/healthchecks_azurekeyvault_secrets_ci.yml
+     - Directory.Build.props
+     - Directory.Build.targets
+   tags-ignore:
+     - release-*
+     - preview-*
+
+  pull_request:
+    branches: [ master ]
+    paths:
+      - src/HealthChecks.AzureKeyVault.Secrets/**
+      - test/HealthChecks.AzureKeyVault.Secrets.Tests/**
```

Comment:
```suggestion
      - src/HealthChecks.Azure.KeyVault.Secrets/**
      - test/HealthChecks.Azure.KeyVault.Secrets.Tests/**
```

Reply:
Yeap sorry, i'm doing it before yours suggestion :-)

Reply:
Hi @adamsitnik  

If it is ok, i will release a preview version to test it and the release if is working well from your side!

Reply:
@unaizorrilla Of course it's OK!